darwin/future/data_objects/properties.py

```diff
@@ -49,14 +71,37 @@ class FullProperty(DefaultDarwin):
 
     id: Optional[str]
     name: str
-    type: str
+    type: PropertyType
     description: Optional[str]
     required: bool
     slug: Optional[str]
     team_id: Optional[int]
     annotation_class_id: Optional[int]
-    property_values: Optional[List[PropertyOption]]
-    options: Optional[List[PropertyOption]]
+    property_values: Optional[List[PropertyValue]]
+    options: Optional[List[PropertyValue]]
+
+    def to_create_endpoint(
+        self,
+    ) -> dict:
+        if not getattr(self, "annotation_class_id") or self.annotation_class_id is None:
```

Comment:
rest of the fields are non-optional so they'll already be captured by ValidationErrors